code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -1041,31 +965,29 @@
 /obj/item/storage/box/emptysandbags
 	name = "box of empty sandbags"
 
-/obj/item/storage/box/emptysandbags/New()
-	..()
-	contents = list()
-	for(var/i in 1 to 7)
+/obj/item/storage/box/emptysandbags/populate_contents()
+	for(var/I in 1 to 7)
 		new /obj/item/emptysandbag(src)
 
 /obj/item/storage/box/rndboards
 	name = "the Liberator's legacy"
 	desc = "A box containing a gift for worthy golems."
 
-/obj/item/storage/box/rndboards/New()
-	..()
-	contents = list()
+/obj/item/storage/box/rndboards/populate_contents()
 	new /obj/item/circuitboard/protolathe(src)
 	new /obj/item/circuitboard/destructive_analyzer(src)
 	new /obj/item/circuitboard/circuit_imprinter(src)
 	new /obj/item/circuitboard/rdconsole/public(src)
 
+/obj/item/storage/box/stockparts
+	display_contents_with_number = TRUE
```

Comment:
sneaky change

Reply:
Yep include that in the PR CL and description

Reply:
Honestly I've never even seen that box before, it just looks nicer with everything counted together